### DISCUSSION TITLE
Multi drop calculation error

### DISCUSSION THREAD

**Discussion by GNSamuelGao** [OpenRadioss/OpenRadioss#2415] [Q&A] [answered]
Hi
I make a multi drop calculation, which means there are several engine files. But only the first engine file is calculated. How to solve this? Below pls check my model file.
[0314_1749.zip](https://github.com/OpenRadioss/OpenRadioss/files/14611233/0314_1749.zip)

**Comment by MarianBulla** [comment, marked as answer]
Hi @GNSamuelGao ,
thanks for your interest in OpenRadioss :-)
Great to having you here!

Regarding your question: You have everything needed: Multiple engine files (*_0001.rad ; *_0002.rad ; _0003.rad).
After lounching the starter with your model file (something like >: starter_win64.exe -i Drop_0000.rad)
You just need to execute the engine (only the engine), inputing the engine files.


Something like:

engine_win64.exe -i Drop_0001.rad
engine_win64.exe -i Drop_0002.rad
engine_win64.exe -i Drop_0003.rad

(No need to launch the starter, again.)

Or you use the nice graphical tool, created by Paul and Hotaosa, which will do it automatically :-) 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/105412502/13cf70ba-5e0c-4ae6-92fe-8aba4cd9fe41)

Hope it helps you ?

Thanks and best regards,
Marian

**Reply by GNSamuelGao** [reply]
Hi Marian,

Got it. Very helpful. :)
